editor/editor_plugin.cpp

```diff
@@ -450,7 +450,7 @@ void EditorPlugin::_bind_methods() {
 	gizmo.return_val.hint = PROPERTY_HINT_RESOURCE_TYPE;
 	gizmo.return_val.hint_string = "EditorSpatialGizmo";
 	ClassDB::add_virtual_method(get_class_static(), gizmo);
-	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "get_name"));
+	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "_get_name"));
```

Comment:
But `_get_name` is virtual, and most virtual methods I see begin with underscore. I am fine with `get_plugin_name` though, so I don't really mind.